src/search/task_utils/variable_order_finder.h

```diff
@@ -24,7 +28,7 @@ extern void dump_variable_order_type(VariableOrderType variable_order_type);
   least as long as the variable order finder.
 */
 class VariableOrderFinder {
-    TaskProxy task_proxy;
+    const TaskProxy &task_proxy;
```

Comment:
I had to look this up :-), but our conventions recommend against a reference here (http://www.fast-downward.org/ForDevelopers/CodingConventions).